array/DNAm/preprocessing/QCwithinCellType.rmd

```diff
@@ -18,21 +18,42 @@ library(pheatmap)
 library(pander)
 library(RColorBrewer)
 library(kableExtra)
+library(data.table)
 
 
-source(args[3]) ### change the content of this file to run QC with different thresholds
-
 dataDir <- args[2]
 setwd(dataDir) 
 
+refDir <-args[3]
 
 gdsFile <-paste0(dataDir, "/2_gds/raw.gds")
+configFile <- paste0(dataDir, "/config.r")

```

Comment:
Need to note somewhere that this change also assumes a new location for the config file